src/index.md

```diff
@@ -36,6 +36,7 @@ topics/index.md
 tutorials.md
 faq.md
 LICENSE.md
+Community.md
```

Comment:
Might be better to use `community.md` to keep things homogeneous, as other files like `tutorials.md` and `faq.md` also start with lower case letter.